recipes/raysect/recipe.yaml

```
@@ -0,0 +1,62 @@
+context:
+  name: raysect
+  version: "0.8.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/${{ name|lower }}/source/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 6b47f06cf58e6bf021606aa3a7b42f0a200f63d8695e01a6274941dedf3685bb
+
+build:
+  script: ${{ PYTHON }} -m pip install --no-build-isolation --no-deps -vv .
+  number: 0
+  skip:
+    - win
+
+requirements:
+  build:
+    - if: unix
+      then:
+        - ${{ compiler("c") }}
+    - if: win
+      then:
+        - ${{ compiler("clang") }}
+    - ${{ stdlib("c") }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython <3.0
+    - numpy <2.0
```

This should use the global pin
```suggestion
    - numpy
```

---

Should I remove the version specification?

---

Yes

---

But, how should I set the `numpy <2.0` limitation in runtime?

---

Ah, yes, I can see it!
Additionally, at the default [`conda_build_config.yaml` ](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml), `numpy` version is specified at each python one.